index.html

```diff
@@ -332,17 +354,20 @@ <h1 class="contact-me">Contact me</h1>
             placeholder="email"
           />
           <textarea
-            id="messag"
+            id="message"
             class="text-area"
             placeholder="Write your message"
```

Comment:
kindly this placeholder is not clear. try to fix it in your css file. remember you should follow the figma template design with the correct color, text...